### DISCUSSION TITLE
In prompt: is there a way to get the answer to a previous prompt as a value in transform or default?

### DISCUSSION THREAD

**Discussion by seanonthenet** [c4spar/cliffy#309] [Q&A]
I'm trying to set the value of a prompt based on a previous prompt's answer. I know I can use `before` or `after` methods but I don't see a way of extracting previous answers from them and using them in the current prompt. 

Thanks for any help :) :)

**Comment by c4spar** [comment]
Hi @seanonthenet, sry for late reply!

No, it's currently not possible, but it is something we should implement.

**Reply by seanonthenet** [reply]
Ok thanks. Should I log issue or anything?

**Reply by c4spar** [reply]
That would be great 👍 thx!
